Tranga/MangaConnectors/MangaKatana.cs

```diff
@@ -167,9 +167,9 @@ private List<Chapter> ParseChaptersFromHtml(Manga manga, string mangaUrl)
 
 		HtmlNode chapterList = document.DocumentNode.SelectSingleNode("//div[contains(@class, 'chapters')]/table/tbody");
 
-		Regex volumeRex = new(@"Volume ([0-9]+)");
-		Regex chapterNumRex = new(@"https:\/\/mangakatana\.com\/manga\/.+\/c([0-9\.]+)");
-		Regex chapterNameRex = new(@"Chapter [0-9\.]+: (.*)");
+		Regex volumeRex = new(@"(Volume ([0-9]+))|(Vol\.[0-9]+)");
+		Regex chapterNumRex = new(@"https:\/\/mangakatana\.com\/manga\/.+\/(c([0-9\.]+))|(v[0-9]+c[0-9]+)");
+		Regex chapterNameRex = new(@"Chapter [0-9\.]+:? (.*)");
```

Comment:
Nice Regex, but I noticed a better way:
We can also parse the volume number from the url (like with chapters).
Would be better to just have a single Regex that can fetch both groups.
`https://mangakatana.com/manga/heavens-door-koike-keiichi.13112/c10` Chapter would be 10, Volume 1
`https://mangakatana.com/manga/ultra-heaven.10029/v3c2` Chapter would be 2, Volume 3

Reply:
Problem is that mangakatana seems to be a bit inconsistent with the namings in the url. Found at least 5 variants. But dusted off the little regex skills i have and this should cover the ones i have seen